docs/architecture/page.rst

```diff
@@ -0,0 +1,41 @@
+Saleor Pages
```

Comment:
Static pages

Reply:
Are "static pages" a good name? They are not really static (as you can edit them in the dashboard) and even if they were the name focuses on the means rather than a goal (what problems do they solve?).